docs/run-mock-test.md

```diff
@@ -33,7 +33,7 @@ Install Docker compose: https://docs.docker.com/compose/install/
 ```
 cd aws-otel-test-framework/terraform/mock
 terraform init
-terraform apply -var="testcase=../testcases/otlp_mock" -var-file=../testcases/otlp_mock/parameters.tfvars
+terraform apply -var="testcase=../testcases/otlp_mock" -var-file="../testcases/otlp_mock/parameters.tfvars"
```

Comment:
Is this actually a syntax issue? Doesn't the command still work without the quotes? I'm not opposed to updating the docs to include the quotes.

Reply:
looks like the command still works, its more of like a good practice/consistent syntax to include the quotes.